pygrametl/__init__.py

```diff
@@ -444,23 +444,38 @@ def now(ignoredtargetconn=None, ignoredrow=None, ignorednamemapping=None):
 
 
 def ymdparser(ymdstr):
-    """Convert a string of the form 'yyyy-MM-dd' to a datetime.date.
+    """Convert input of the form 'yyyy-MM-dd' to a datetime.date.
```

Comment:
_"input of the form 'yyyy-MM-dd'"_ implies that the input must be string, maybe we could write something like _"Convert a string of the form 'yyyy-MM-dd', a datetime.datetime, or a datetime.date to a datetime.date."_ Although, that still does not mention that `None` is an acceptable input.

Reply:
It said "string" before this change, but now the function accepts more - and also more than string, datetime.datetime, and datetime.date. If it gets something else (something in Connection-dependent formats), it will try to cast it to str, but the string should then be in the yyyy-MM-dd form. That is why I changed "string" to "input" here, but I'm fine with another formulation if we can find one that covers it all

Reply:
The part I find confusing is that the text explicitly states that _"input of the form 'yyyy-MM-dd'"_ can be given, but that is only relevant if a string is given as "_'yyyy-MM-dd'"_ has no relation to `datetime.datetime` or `datetime.date` which is also included in _"input"_.

Reply:
Changed to a more precise description